src/material-experimental/mdc-button/fab.ts

```diff
@@ -49,6 +52,8 @@ export class MatFabButton extends MatButtonBase {
   // The FAB by default has its color set to accent.
   color = 'accent' as ThemePalette;
 
+  extended: boolean;
```

Comment:
Is "extended" the only variation or do we expect more? If we expect more, we can turn it into a string instead, e.g. `appearance: 'extended'`.

Reply:
I think it should be the only variation

Reply:
can you make this a setter/getter and use `coerceBooleanInput` that way peopel can do `<button mat-fab extended>` instead of `<button mat-fab [extended]="true">` (`coerceBooleanInput` does some non-standard type coercion, e.g. `'' => true`)